pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -203,6 +211,18 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 		}
 	}
 
+	// Since we will most likely get N different timestamps, each with frequency=1, we get the median instead of the mode.
+	sortkeys.Uint32s(sortedTimestamps)
+	timestampCount := len(sortedTimestamps)
+	mid := timestampCount / 2
+	if timestampCount%2 == 1 {
+		finalTimestamp = sortedTimestamps[mid]
+	} else {
+		finalTimestamp = (sortedTimestamps[mid-1] + sortedTimestamps[mid]) / 2
+	}
+
+	fmt.Println("finalTimestamp: ", finalTimestamp)
```

Comment:
log, not print

Reply:
This is an unintended log, I'll remove it.